### ISSUE TITLE
RTF: list of classes in Namespace Documentation not separated CRLF (Origin: bugzilla #741547)

### ISSUE THREAD

**Issue by doxygen** [doxygen/doxygen#5691]
status NEW severity *normal* in component *general* for *---*
Reported in version *1.8.8* on platform *Other*
Assigned to: Dimitri van Heesch

Original attachment names and IDs:
- _[doxytest.zip](https://sourceforge.net/projects/doxygen/files/attachments/bug741547_292746_doxytest.zip/download/download)_ (ID 292746)
- _[PreferrableResult.zip](https://sourceforge.net/projects/doxygen/files/attachments/bug741547_372450_PreferrableResult.zip/download/download)_ (ID 372450)
- _[bug_741547_1_8_14.zip](https://sourceforge.net/projects/doxygen/files/attachments/bug741547_372460_bug_741547_1_8_14.zip/download/download)_ (ID 372460)

On 2014-12-15 10:22:15 +0000, t-sano@cp.jp.nec.com wrote:

    Created attachment 292746
    test data for reproducing this problem
    
    I had the following result in rtf:
    =======
    Namespace Documentation
    
    Package a
    
    Classes
    * class Test
    * class for test class Test2
    Test2 summary  
    =======
    
    This will be fine:
    ======
    * class Test
      class for test
    * class Test2
      Test2 summary
    ======
    
    Also, In the "File Reference" section, the paragraph title "Namespaces" does
    not begin at the line start.

On 2015-01-05 14:41:44 +0000, Frank HJ Cuypers wrote:

    I have the same issue. Itseems that the brief description of a class is put
    on the same bullet as the name of the next class.

On 2018-05-26 17:57:21 +0000, albert wrote:

    With version 1.8.8 I see the output as indicated.
    With version 1.8.14 I see the correct output
    
     I think the problem has been solved in the mean time.

On 2018-05-28 13:18:35 +0000, albert wrote:

    Looks like problem has been solved by means of pull request 596,
    https://github.com/doxygen/doxygen/pull/596.
    
    I just pushed a more general solution undoing again pull request 746: pull
    request 717, https://github.com/doxygen/doxygen/pull/746

On 2018-05-28 23:40:51 +0000, t-sano@cp.jp.nec.com wrote:

    Created attachment 372450
    1.8.14 execution results on "test data for reproducing this problem"

On 2018-05-28 23:46:41 +0000, t-sano@cp.jp.nec.com wrote:

    Thank you for kind notices.
    I downloaded doxygen-1.8.14-setup.exe (44.4MB) ( http ) from
    http://www.stack.nl/~dimitri/doxygen/download.html
    and install it on Windows7 Professional.
    Using "test data for reproducing this problem", I got an RTF file that seems
    not fixed. Please seee attachment 372450. The screenshot was taken on
    wordpad. Microsoft Word 2013 produces similar result except that "Class
    Documentation" locates the top of the page and separated from "Test2
    summary".

On 2018-05-29 08:21:24 +0000, albert wrote:

    Created attachment 372460
    Results with the 1.8.14 version
    
    I had another look with the 1.8.14 version and it looks all OK to me.
    The attachment contains:
    - print of WordPad (although WordPad is not really suitable, see e.g.
    strings like AAAAAAAAACAAAAAAAAAC and page divisions), though texts look OK
    to me
    - print of the output in word (a bit small) but looks OK to me as well
    - the refman.rtf file I got.
    
    Can you please check what you do see with my rtf file.
    Some other steps:
    - enable HTML and see which version is mentioned (at the bottom of the
    (page)
    - attach your rtf file to this bug report.

**Comment by tateki** [comment]
Please see the attached file: 
[refman.zip](https://github.com/doxygen/doxygen/files/2240930/refman.zip)
I create it on Windows7 Professional. The doxygen version is 1.8.14 (seen at the bottom of the page).
I try Win10 Pro, then get correct result. Because Win7 will come to its end on Jan. 2020, I don't think this issue is important. Thank you.

**Comment by albert-github** [comment]
Can you please be a but more specific where I should look in the refman.rtf / indicate what is exactly wrong.
I assume the input is still the same as indicated in the original bug report.

**Comment by tateki** [comment]
Please see Page 5 "Namespace Documentation" "a Namespace Reference" (8/13 page)
I copied the bad result on Win7 Pro below. Win10 result is located in pale orange rectangle.
![phenomenon](https://user-images.githubusercontent.com/29248554/43449157-b53f8c38-94ea-11e8-91d7-529292d8ba09.png)

**Comment by albert-github** [comment]
I ran the original problem from 'doxytest.zip' with version 1.8.14 on windows 7 and the results look OK to me.  When looking at 'refman.zip' you attached I see also that it does not look OK.
Please attach the case you used (and also double check the version of doxygen on Windows 7, doxygen -v or enable HTML output and see which version it mentions at the bottom of the page).

**Comment by tateki** [comment]
First, I use very old environment: Win7Pro, Office 2013.
I created the bad result from doxytest.zip (ID 292746) as:
1. Unzip doxytest.zip
2. run doxygen as
DOS> doxygen Doxyfile

I got the following output:
D:\ProgramData\doxygen\20180731>doxygen -v
1.8.14

D:\ProgramData\doxygen\20180731>doxygen Doxyfile
warning: the dot tool could not be found at C:\Program Files\Graphviz2.38\bin
warning: failed to open layout file 'DoxygenLayout.xml' for reading!
Searching for include files...
Searching for example files...
Searching for images...
Searching for dot files...
Searching for msc files...
Searching for dia files...
Searching for files to exclude
Searching INPUT for files to process...
Searching for files in directory D:/ProgramData/doxygen/20180731/input
Reading and parsing tag files
Parsing files
Preprocessing D:/ProgramData/doxygen/20180731/input/test.cs...
Parsing file D:/ProgramData/doxygen/20180731/input/test.cs...
Preprocessing D:/ProgramData/doxygen/20180731/input/test2.cs...
Parsing file D:/ProgramData/doxygen/20180731/input/test2.cs...
Building group list...
Building directory list...
Building namespace list...
Building file list...
Building class list...
Associating documentation with classes...
Computing nesting relations for classes...
Building example list...
Searching for enumerations...
Searching for documented typedefs...
Searching for members imported via using declarations...
Searching for included using directives...
Searching for documented variables...
Building interface member list...
Building member list...
Searching for friends...
Searching for documented defines...
Computing class inheritance relations...
Computing class usage relations...
Flushing cached template relations that have become invalid...
Computing class relations...
Add enum values to enums...
Searching for member function documentation...
Creating members for template instances...
Building page list...
Search for main page...
Computing page relations...
Determining the scope of groups...
Sorting lists...
Freeing entry tree
Determining which enums are documented
Computing member relations...
Building full member lists recursively...
Adding members to member groups.
Computing member references...
Inheriting documentation...
Generating disk names...
Adding source references...
Adding xrefitems...
Sorting member lists...
Computing dependencies between directories...
Generating citations page...
Counting data structures...
Resolving user defined references...
Finding anchors and sections in the documentation...
Transferring function references...
Combining using relations...
Adding members to index pages...
error: Can't open RTF style sheet file rtfstyle.cfg. Using defaults.
Generating style sheet...
Generating search indices...
Generating example documentation...
Generating file sources...
Parsing code for file test.cs...
Parsing code for file test2.cs...
Generating file documentation...
Generating docs for file test.cs...
Generating docs for file test2.cs...
Generating page documentation...
Generating group documentation...
Generating class documentation...
Generating namespace index...
Generating docs for namespace a
Generating docs for compound a::Test...
Generating docs for compound a::Test2...
Generating graph info page...
Generating directory documentation...
Generating index page...
Generating page index...
Generating module index...
Generating namespace index...
Generating namespace member index...
Generating annotated compound index...
Generating alphabetical compound index...
Generating hierarchical class index...
Generating member index...
Generating file index...
Generating file member index...
Generating example index...
finalizing index lists...
writing tag file...
Combining RTF output...
lookup cache used 4/65536 hits=8 misses=4
finished...

D:\ProgramData\doxygen\20180731>
